edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/DependencyManager.cs

```diff
@@ -151,6 +151,7 @@ void RegisterMqttModule(ContainerBuilder builder, (bool isEnabled, bool usePersi
             bool encryptTwinStore = this.configuration.GetValue("EncryptTwinStore", false);
             int configUpdateFrequencySecs = this.configuration.GetValue("ConfigRefreshFrequencySecs", 3600);
             TimeSpan configUpdateFrequency = TimeSpan.FromSeconds(configUpdateFrequencySecs);
+            bool closeCloudConnectionWhenCloseDeviceConnection = this.configuration.GetValue("CloseCloudConnectionWhenCloseDeviceConnection", true);
```

Comment:
Looks like my suggestion got overwritten? 
```suggestion
            bool closeCloudConnectionWhenCloseDeviceConnection = this.configuration.GetValue("CloseCloudConnectionOnDeviceDisconnect", true);
```